sdk/src/main/java/io/radar/sdk/RadarApiHelper.kt

```diff
@@ -48,6 +54,9 @@ internal open class RadarApiHelper(
                 urlConnection.requestMethod = method
                 urlConnection.connectTimeout = 10000
                 urlConnection.readTimeout = 10000
+                if (stream) {
+                    urlConnection.setChunkedStreamingMode(1024)
```

Comment:
should this be a const at the top of this file? same with the conn / read timeouts above?

Reply:
My preference would be not to make this change at this time, but let's discuss offline